packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -402,5 +409,8 @@ export const schemaFactory = (api?: Api) => {
         .required(),
       channelCashoutsEnabled: Yup.boolean(),
     }),
+    updatePalletFrozenStatus: Yup.object().shape({
+      frozen: Yup.boolean().required('Field is required'),
```

Comment:
This is a bit tricky but it's really important that the proposal clarify things here by having an "execution" constraint. It means that if the `ProjectToken` pallet is currently enabled and someone try to enable it, they should get a warning warning telling them that the pallet is already enabled and that the proposal would fail if executed right now.

To do that you can pass "execution" as the validation name. There are already a few examples in this file.